CMSgov/mint-app
# [NOREF] Set platform property for some images that require linux/amd64

# NOREF

## Changes and Description

- Adds `platform: linux/amd64` to some images that put out warnings when spinning up the docker-compose stack.

![image](https://user-images.githubusercontent.com/15203744/213213073-21984754-e6ae-471c-9375-c1db7f4aec3c.png)

## How to test this change

1. Run `scripts/dev up`
2. Ensure no warnings are output to the console about not specifying a platform type.
3. Ensure the application runs as expected

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
